app/src/main/java/com/droidhats/mapprocessor/AStar.kt

```diff
@@ -117,37 +117,18 @@ fun findNearestPoint(mapElement: MapElement, pathElements: List<Vertex>): Vertex
 
 /**
  * Takes 2 map elements and returns the shortest path in between them given the list of path elements
+ * as an arrow
  * @param start element
  * @param end element
  * @param list of vertices
  * @return string
```

Comment:
Can we add more details to the parameters of this method. The return is not needed since the method description covers it.